credentials/alts/internal/handshaker/handshaker_test.go

```diff
@@ -102,6 +106,18 @@ func (t *testRPCStream) Send(req *altspb.HandshakerReq) error {
 			}
 		}
 	} else {
+		switch req := req.ReqOneof.(type) {
+		case *altspb.HandshakerReq_Next:
+			// Compare the network_latency_ms field to the minimum expected network
+			// latency.
+			networkLatency := time.Duration(req.Next.NetworkLatencyMs) * time.Millisecond
+			if networkLatency < t.minExpectedNetworkLatency {
+				return fmt.Errorf("networkLatency (%v) is smaller than expected min network latency (%v)", networkLatency, t.minExpectedNetworkLatency)
+			}
```

Comment:
Optional: This could be shortened as:

```golang
if nl := time.Duration(req.Next.NetworkLatencyMs) * time.Millisecond; nl < t.minExpectedNetworkLatency {
	return fmt.Errorf("networkLatency (%v) is smaller than expected min network latency (%v)", nl, t.minExpectedNetworkLatency)
}
```

Prefer `nl` over `networkLatency` given how small the scope of this variable is. See: https://google.github.io/styleguide/go/decisions#variable-names

Reply:
Done.